### PR TITLE
[FIX] when bank name is too long, it overlaps with 'giro' bank

### PR DESCRIPTION
**Pull request by eLBati**
See example with 'Banca Raiffeisen del Cassarate':

bvr

### CODE REVIEW

File: l10n_ch_payment_slip/payment_slip.py [hunk 1]
```diff
@@ -547,7 +548,8 @@ def _draw_bank(self, canvas, print_settings, initial_position, font, bank):
         text = canvas.beginText()
         text.setTextOrigin(x, y)
         text.setFont(font.name, font.size)
-        lines = bank.name.split("\n")
+        bank_name = textwrap.fill(bank.name, 26)
```
Comment (eLBati) [inline l10n_ch_payment_slip/payment_slip.py:551]: @nbessi is 26 always valid? Should it be parametric?

Comment (yvaucher) [inline l10n_ch_payment_slip/payment_slip.py:551]: This font seems monospaced. So it should be ok.

**Comment by yvaucher** [comment]
:+1:

**Comment by nbessi** [comment]
:+1:

**Pull request merged**